src/shared/codelens/codeLensUtils.ts

```diff
@@ -66,7 +67,11 @@ export async function makeCodeLenses({
                 rootUri: handler.manifestUri,
                 runtimeFamily,
             }
-            if (!existingConfigs.has(path.join(path.dirname(baseParams.rootUri.fsPath), baseParams.handlerName))) {
+            if (
+                !existingConfigs.has(
+                    pathutils.normalize(path.join(path.dirname(baseParams.rootUri.fsPath), baseParams.handlerName))
```

Comment:
nit: Recommend moving this change to separate PR if not related to this one.

Reply:
I forget what this was for, might have been for upload Lambdas (and I forgot to swap the branch). I'll give it a look.